CONTRIBUTING.md

```diff
@@ -40,14 +40,23 @@ Guidelines](https://opensource.google.com/conduct/).
 
 ### Option 1: Running locally
 
-> 💡 Recommended if you're planning to develop the application.
+> 💡 Recommended if you're planning to develop the application.  
+<<<<<<< HEAD
+> ℹ️ Prerequisite: [Install Cloud SDK](https://cloud.google.com/sdk/docs/quickstart)
+=======
+> ℹ️  Prerequisite: [Cloud SDK should be installed.](https://cloud.google.com/sdk/docs/quickstart)  
+>>>>>>> master
```

Comment:
Looks like you have a bad merge: <<<HEAD is not supposed to be in the file

Reply:
Merge mix up - resolved